### ISSUE TITLE
Automatic Export/Backup

### ISSUE THREAD

**Issue by j-stechmann** [Demizo/Daily_You#93]
It would be awesome if the app could automatically export its data to a folder, on a schedule of once a week (or some other time period).

I like syncing all my phone app backups regularly using Syncthing, so if I could just point Syncthing at a folder and have it pull backups regularly, it would save me a lot of work.

**Comment by Demizo** [comment]
Currently, the app lets you set external folders for app data. While not a real backup, it does allow you to store the app data in a synced folder. Effectively this is an instant single backup. 

That said, I like the idea of automatic backups. I'm just not sure if it makes sense to have both features. I'm open to thoughts on this.

**Comment by BlizzardWizzard** [comment]
Thats true, I didn't think about simply setting the app data location to something Syncthing can grab.
That might even be a more storage efficient way of doing it.

I was thinking about the export logs/images function, as in maybe set up the export location once and have the app export the logs and images on a schedule.

The warning when changing the app data location says that the contents of the chosen folder would overwrite the current data, which indicates importing of the data is supported - which is all I really need?

I think that means I'm happy with the current state of things, I just didn't realize the potential of this function from the in-app description.

Thank you so much 😄

**Comment by druivensap** [comment]
I would also like to see an automatic backup function
Only thing I'm missing in this amazing app

**Comment by Demizo** [comment]
@druivensap Agreed, in the meantime, if you haven't done so already, the Storage settings let you sync with an external folder. That would at least give you a 1:1 up to date mirror as a backup.

**Comment by Berrigold** [comment]
I would really like this feature too, I'd love to be able to sync it between my devices using syncthing.

**Comment by Demizo** [comment]
@Berrigold If you want syncing, the storage settings let you set an external folder to sync with, you can then sync that folder across devices with syncthing. The feature is more hidden than it should be.

**Comment by Berrigold** [comment]
Thank you @Demizo, so this allows for automatic backup/sync then? That would be wonderful, thank you.

**Comment by Demizo** [comment]
@Berrigold Yep, sorta! The app will sync any changes to that folder as you make them. So more of a mirror than a backup. But with syncthing you can back it up whereever.

**Comment by Berrigold** [comment]
@Demizo wonderful, this will work! Thank you!
